src/locales/de/messages.json

```diff
@@ -806,6 +806,12 @@
   "enableMinToTrayDesc": {
     "message": "Zeige ein Symbol in der Taskleiste, wenn das Fenster minimiert wird."
   },
+  "enableCloseToTray": {
```

Comment:
revert this file. only English is added during development and translations are done via crowdin.